### DISCUSSION TITLE
Tofu 0.10.3

### DISCUSSION THREAD

**Discussion by Odomontois** [tofu-tf/tofu#707] [General]
## 🚀 Features

- Continuational representation of ConnectionIO @catostrophe (#698)
- [ZIO] Unlift Part of big Has Context @Odomontois (#694)
- [Config] Allow simple types to be parsed from strings @Odomontois (#693)
- Context typeclasses @FunFunFine (#687)
- Move Data to kernel, remove data module @Odomontois (#685)
- Move Scoped to kernel @Odomontois (#684)
- Kernelize Start\Fire @Odomontois (#682)
- move timeout and guarantee syntax to kernel @Odomontois (#683)
- Guarantee kernelize @Odomontois (#681)
- Timeout moved to kernel @Odomontois (#680)
- kernelify Delay via optional CE2 interop @Odomontois (#678)
- Minimal Kernel @Odomontois (#675)
- Display  @FunFunFine (#450)
- `evalFilter` combinator. @oskin1 (#632)

## 🐛 Bug Fixes

- Apply masking to logged fields @Odomontois (#704)

## 🧰 Maintenance

- Use kernel dependency for derivation, zioCore and streams modules. @FunFunFine (#705)
- Context remove prepare 3.0 @FunFunFine (#702)
- Update fs2-io to 2.5.9 @scala-steward (#696)
- Update sbt to 1.5.5 @scala-steward (#699)
- Update sbt-scalafmt to 2.4.3 @scala-steward (#690)
- Update scala-collection-compat to 2.5.0 @scala-steward (#691)
- Rework Build @Odomontois (#689)
- Update fs2-io to 2.5.8 @scala-steward (#686)
- Update sbt to 1.5.4 @scala-steward (#670)
- Update slf4j-api to 1.7.31 @scala-steward (#671)
- Update enumeratum to 1.7.0 @scala-steward (#672)
- Update fs2-io to 2.5.7 @scala-steward (#676)
- Update derevo-cats-tagless, derevo-core to 0.12.6 @scala-steward (#677)
- Bump olafurpg/setup-scala from 10 to 12 @dependabot (#668)
- Update zio to 1.0.9 @scala-steward (#666)
- Update sbt-tpolecat to 0.1.20 @scala-steward (#665)
- Update sbt to 1.5.3 @scala-steward (#664)
- Update scala-library, scala-reflect to 2.12.14 @scala-steward (#660)
- Bump actions/cache from 2.1.5 to 2.1.6 @dependabot (#662)
- Update sbt-scalafix to 0.9.29 @scala-steward (#661)
- Update tethys-core, tethys-jackson to 0.24.0 @scala-steward (#659)
- Update sbt-tpolecat to 0.1.19 @scala-steward (#658)
- Update circe-core to 0.14.1 @scala-steward (#657)
- Update refined to 0.9.26 @scala-steward (#656)
- Update doobie-core to 0.13.4 @scala-steward (#654)
- Update sbt-git to 1.0.1 @scala-steward (#652)
- Update zio-interop-cats to 2.5.1.0 @scala-steward (#649)
- Update scala-library, scala-reflect to 2.13.6 @scala-steward (#645)
- Update log4cats-core to 1.3.1 @scala-steward (#646)
- Update doobie-core to 0.13.3 @scala-steward (#647)
- Update zio to 1.0.8 @scala-steward (#648)
- Update doobie-core to 0.13.2 @scala-steward (#629)
- Update sbt to 1.5.2 @scala-steward (#631)
- Update alleycats-core, cats-core, cats-free to 2.6.1 @scala-steward (#634)
- Update cats-effect to 2.5.1 @scala-steward (#635)
- Update cats-mtl to 1.2.1 @scala-steward (#636)
- Update monix to 3.4.0 @scala-steward (#637)
- Update scala-collection-compat to 2.4.4 @scala-steward (#638)
- Update fs2-io to 2.5.6 @scala-steward (#639)
- Update refined to 0.9.25 @scala-steward (#640)
- Update sbt-mdoc to 2.2.21 @scala-steward (#641)
- Update scalatest to 3.2.9 @scala-steward (#642)
- Update kind-projector to 0.13.0 @scala-steward (#643)
- Update shapeless to 2.3.7 @scala-steward (#644)
- Update shapeless to 2.3.5 @scala-steward (#628)

<hr /><em>This discussion was created from the release <a href='https://github.com/tofu-tf/tofu/releases/tag/v0.10.3'>Tofu 0.10.3</a>.</em>

**Comment by catostrophe** [comment]
Congrats
